x-pack/elastic-agent/pkg/agent/cmd/enroll_cmd.go

```diff
@@ -323,6 +323,24 @@ func (c *enrollCmd) prepareFleetTLS() error {
 	return nil
 }
 
+func (c *enrollCmd) daemonReloadWithBackoff(ctx context.Context) error {
+	err := c.daemonReload(ctx)
```

Comment:
If no error here then no reason to continue. Just return as it has been reloaded.